src/snapshot_middleware/project.rs

```diff
@@ -603,6 +576,29 @@ fn project_node_should_reserialize(
         }
     }
 
+    let db = rbx_reflection_database::get();
+    if let Some(class_descriptor) = db.classes.get(instance.class_name()) {
+        for (prop_name, inst_value) in instance.properties() {
+            if prop_name == "Attributes" {
+                continue;
+            }
+            let Some(default_value) = db.find_default_property(class_descriptor, prop_name) else {
+                continue;
+            };
```

Comment:
If a default property cannot be found, is "don't reserialize" the correct behavior?